card/mod/standard/set/all/rich_html/menu.rb

```diff
@@ -22,7 +22,8 @@
   end
 
   def menu_path_opts
-    opts = { slot: { home_view: voo.home_view } }
+    opts = { slot: { home_view: (voo.home_view || @slot_view),
```

Comment:
the menu links are a bit special. The gear has its own slot (because the menu is loaded via ajax) which means the menu links update a different slot. I'm not sure if the slotter class can handle that.